### ISSUE TITLE
TypeError: window.localStorageFallback.setItem is not a function

### ISSUE THREAD

**Issue by joelabair** [ripeworks/local-storage-fallback#4]
When running your Browser Bundle example from README.md - I get the following;
``` Uncaught TypeError: window.localStorageFallback.setItem is not a function ```

Seems that the example code may be off.  This works though.
```
<script src="https://unpkg.com/local-storage-fallback/lib/dist.min.js"></script>
<script>
  window.localStorageFallback.default.setItem('foo', 'bar')
</script>
```

**Comment by joelabair** [comment]
Upon deeper inspection - this may be due to a missing babel plugin.  babel-plugin-add-module-exports restores the build to function as documented.

**Comment by tamagokun** [comment]
@joelabair this should be fixed in `1.4.0` Thanks!
